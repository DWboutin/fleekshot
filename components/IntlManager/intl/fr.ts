import flat from "flat";

import signUpForm from "../../SignUpForm/intl/fr";
import signInForm from "../../SignInForm/intl/fr";
import profile from "../../Profile/intl/fr";
import { IntlMessages } from "./types";

const fr = {
  home: "Accueil",
  ...signUpForm,
  ...signInForm,
  ...profile,
};

export default flat(fr, { delimiter: "_" }) as IntlMessages;
