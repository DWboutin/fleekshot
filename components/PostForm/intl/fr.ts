import { PostFormIntl } from "./type";

const fr: PostFormIntl = {
  postForm: {
    uploadImage: "Téléverser une image",
    share: "Partager",
    inputLabel: "Écrivez votre prochaine publication",
    errors: {
      tooShort: "Est trop court",
      tooLong: "Est trop long",
      isRequired: "Est requis",
      imageIsRequired: "Une image est requis",
    },
  },
};

export default fr;
