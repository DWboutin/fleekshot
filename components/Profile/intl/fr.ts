import { ProfileIntl } from "./type";

const fr: ProfileIntl = {
  profile: {
    title: "Profil",
    uploadPicture: "Téléverser",
  },
};

export default fr;
