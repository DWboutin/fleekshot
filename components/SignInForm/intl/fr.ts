import { SignInFormIntl } from "./type";

const fr: SignInFormIntl = {
  signInForm: {
    input: {
      username: {
        label: "Nom D'utilisateur",
      },
      password: {
        label: "Mot de passe",
      },
    },
    button: {
      submit: "submit",
    },
    errors: {
      tooShort: "Trop court",
      tooLong: "Trop long",
      isRequired: "Est requis",
      notStrongEnough: "Trop faible",
      mismatch: "Ne correspond pas",
    },
  },
};

export default fr;
