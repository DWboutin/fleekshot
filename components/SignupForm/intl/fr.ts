import { SignupFormIntl } from "./type";

const fr: SignupFormIntl = {
  signupForm: {
    input: {
      name: {
        label: "Nom complet",
      },
      username: {
        label: "Nom D'utilisateur",
      },
      password: {
        label: "Mot de passe",
      },
      confirmPassword: {
        label: "Confirmation du mot de passe",
      },
    },
    errors: {
      tooShort: "Trop court",
      tooLong: "Trop long",
      isRequired: "Est requis",
      notStrongEnough: "Trop faible",
      mismatch: "Ne correspond pas",
    },
  },
};

export default fr;
