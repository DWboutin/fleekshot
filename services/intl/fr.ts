import { ErrorCodes } from "../../server/handler/errorHandler";
import { HttpErrorsIntl } from "./type";

const fr: HttpErrorsIntl = {
  httpErrors: {
    [ErrorCodes.DuplicateInDatabase]:
      'Le champ "{field}" avec la valeur "{value}" se trouve déjà dans la base de donnée',
    [ErrorCodes.ValidationError]: 'Le champ "{field}" {message}',
    [ErrorCodes.NoUser]: "Erreur de connexion - {message}",
    field: {
      name: "nom complet",
      username: "nom d'utilisateur",
      password: "password",
      confirmPassword: "confirm password",
    },
  },
};

export default fr;
