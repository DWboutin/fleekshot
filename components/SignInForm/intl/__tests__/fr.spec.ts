import fr from "../fr";

describe("intl SignUpForm fr", () => {
  it("should match snapshot", () => {
    expect(fr).toMatchSnapshot();
  });
});
