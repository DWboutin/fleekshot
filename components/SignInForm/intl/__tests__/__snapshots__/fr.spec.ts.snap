// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intl SignUpForm fr should match snapshot 1`] = `
Object {
  "signInForm": Object {
    "button": Object {
      "submit": "submit",
    },
    "errors": Object {
      "isRequired": "Est requis",
      "mismatch": "Ne correspond pas",
      "noUser": "No user was found",
      "notStrongEnough": "Trop faible",
      "tooLong": "Trop long",
      "tooShort": "Trop court",
    },
    "input": Object {
      "password": Object {
        "label": "Mot de passe",
      },
      "username": Object {
        "label": "Nom D'utilisateur",
      },
    },
  },
}
`;
